spec/jobs/send_expired_vacancy_feedback_email_prompt_job_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+RSpec.describe SendExpiredVacancyFeedbackPromptJob do
+  subject(:job) { described_class.perform_later }
+  let(:expiry_date) { 3.weeks.ago }
+  let(:mail) { double("Mail::Message", deliver_later: true) }
+  let(:publisher) { create(:publisher, email: publisher_email) }
+  let(:publisher_email) { "test@example.com" }
+
+  before { allow(Publishers::ExpiredVacancyFeedbackPromptMailer).to receive(:prompt_for_feedback).and_return(mail) }
+
+  context "when the publisher has 5 or less vacancies to be prompted on" do
+    let!(:expired_vacancies) do
+      [create(:vacancy, :expired, publisher: publisher, expires_at: expiry_date),
```

Comment:
why you are not using create_list instead of create?

Reply:
I don't know actually. I think I did at some point but changed it for some reason and perhaps forgot to change it back 🤷‍♂️